scanapi/__init__.py

```diff
@@ -18,14 +18,22 @@
     type=click.Path(exists=True),
     default=SETTINGS["spec_path"],
 )
-@click.option("-d", "--docs-path", "docs_path", default=SETTINGS["docs_path"])
+@click.option("-d", "--docs-path", "docs_path")
+@click.option(
+    "-r",
+    "--reporter",
+    "reporter",
+    type=click.Choice(["console", "markdown", "html"]),
+    default=SETTINGS["reporter"],
+)
+@click.option("-t", "--template", "template")
```

Comment:
Added now but not being used yet.

Reply:
I was thinking in implementing this as part of another PR. And maybe for now call logger saying it’s not implemented yet. What do you think?

Reply:
Hmm i think it is a good idea, I like it

Reply:
```
➜  scanapi-example scanapi --template=bla.html
WARNING:scanapi:Custom templates are not supported yet. Soon to be. Hang tight.
INFO:scanapi.yaml_loader:Loading file api.yaml
INFO:scanapi.tree.api_tree:Building requests
INFO:scanapi.reporter:Writing documentation
INFO:scanapi.reporter:The documentation was generated successfully.
INFO:scanapi.reporter:It is available at docs.md
```